stdlib/cache/src/main/ballerina/src/cache/Module.md

```diff
@@ -28,16 +30,18 @@ public type AbstractEvictionPolicy abstract object {
 };
 ```
 
-The Ballerina Cache module provides a `Cache` object, which is a map data structure based implementation of the `AbstractCache` object. It is not recommended to insert `()` as the value of the cache since it doesn't make any sense to cache a nil. Also, it provides the `LruEvictionPolicy` object, which is based on the LRU eviction algorithm.
+The Ballerina Cache module provides the `cache:Cache` object, which is a `map` data structure based implementation of the `cache:AbstractCache` object. It is not recommended to insert `()` as the value of the cache since it doesn't make any sense to cache a nil. Also, it provides the `cache:LruEvictionPolicy` object, which is based on the LRU eviction algorithm.
```

Comment:
```suggestion
The Ballerina Cache module provides the `cache:Cache` object, which is a `map` data structure based implementation of the `cache:AbstractCache` object. It is not recommended to insert `()` as the value of the cache since it doesn't make sense to cache a nil. Also, it provides the `cache:LruEvictionPolicy` object, which is based on the LRU eviction algorithm.
```